tools/applyconfig/applyconfig.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+type options struct {
+	confirm   bool
+	level     string
+	user      string
+	directory string
+}
+
+var adminConfig = regexp.MustCompile(`^admin_.+\.yaml$`)
+var standardConfig = regexp.MustCompile(`^cfg_.+\.yaml$`)
```

Comment:
doesn't this require a `cfg_` prefix? README says any YAML

Reply:
This regex is not used anymore, I need to remove it.